### PR TITLE
feat(ui): autocomplete for identifier template

### LINKED ISSUE #421: Add the table name to the prefix in the case of an internal identifier (no http(s):// in the uri).
- [ ] In a table if the {Identifier template} is not absolute (no http(s):// in the uri) add the table name between the cube uri and the identifier itself.

- [ ] Show the composed URI below the field. Similar to the one below found in **Edit column mapping**

Screenshot 2021-01-18 at 17 00 58

### PR DESCRIPTION
**Pull request by martinmaillard**
Auto-fill with table name, auto-complete column names and validation for identifier template input.

Fix #421

### CODE REVIEW

File: apis/core/bootstrap/shapes/table.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ const commonProperties = turtle`
     ${sh.path} ${cc.identifierTemplate} ;
     ${sh.minCount} 1 ;
     ${sh.maxCount} 1 ;
-    ${sh.defaultValue} "" ;
```
Comment (martinmaillard) [inline apis/core/bootstrap/shapes/table.ts:30]: For some reason, `dash:editor` was not taken into account when `sh:defaultValue` was specified. Now I have a feeling we'll run into issues without this default value?

Comment (tpluscode) [reply apis/core/bootstrap/shapes/table.ts:30]: Yes, without `sh:default` we would get validation errors unless the user interacts with the field. Not good.

The problem is that the preferred editor would get removed if there is no metadata about it.

Fixed soon by https://github.com/hypermedia-app/shaperone/pull/84.

Comment (martinmaillard) [reply apis/core/bootstrap/shapes/table.ts:30]: 👍 Can wait on the shaperone PR

Comment (martinmaillard) [reply apis/core/bootstrap/shapes/table.ts:30]: I upgraded shaperone but it seems like it still doesn't select the proper editor when creating a new table

Comment (tpluscode) [reply apis/core/bootstrap/shapes/table.ts:30]: Yes, my mistake

**Review by tpluscode — APPROVED** [review]

**Pull request merged**